DWHD Updates

catavolt opened the pull request:
All the DWHD Updates (Better GFX, A lot of Bugfixes, Implementation of new MyRadio Properties)

catavolt committed:
Bugfixes and new animation BHE

Bugfixes and new animation BHE

catavolt committed:
Bugfixes and new GFX

Bugfixes and new GFX

catavolt committed:
Bugfixes and Enhancements

Bugfixes and Enhancements

catavolt committed:
Bugfixes and Enhancements

Bugfixes and Enhancements

catavolt committed:
Bugfixes and Enhancements

Bugfixes and Enhancements

catavolt committed:
Bugfixes

Bugfixes

catavolt committed:
Bugfixes and Enhancements

Bugfixes and Enhancements

catavolt committed:
bugfixes

Bugfixes

catavolt committed:
Bugfixes, Enhancements, new GFX

Bugfixes, Enhancements, new GFX

catavolt committed:
New default images and implementation of new #Radio properties

New default images and implementation of new #Radio properties

catavolt committed:
Changes regarding MyRadio

Changes regarding MyRadio due to changes made by doskabouter ;)

andrewjswan commented on mediaportal/WindowPlugins/GUIVideos/GUIVideoInfo.cs:
I think this file doesn't belong to the DWHD skin, so it should be removed from the Pull Request.

andrewjswan commented on mediaportal/WindowPlugins/GUIVideos/GUIVideoTitle.cs:
I think this file doesn't belong to the DWHD skin, so it should be removed from the Pull Request.

andrewjswan replied:
Unfortunately, this affects all default skins, so it's not related to DWHD. Also, if you look at what's changed in these files, it's impossible. Everything has changed.
To merge the changes correctly, you need a separate pull request that will show the changes made to the file, as well as the changes for all default skins.

catavolt replied:
OK, then do not integrate the changed .cs files.
Will see how we can solve that later ;)

andrewjswan commented on mediaportal/MediaPortal.Base/skin/DefaultWideHD/DialogMenu.xml:
```diff
@@ -93,15 +93,16 @@
       <posX>460</posX>
       <posY>225</posY>
-      <width>1020</width>
+      <width>1020</width> 1
```
```suggestion
      <width>1020</width>
```

catavolt replied:
This is the corrected xml ;) Sorry!!!
[DialogMenu.xml](https://github.com/user-attachments/files/23901319/DialogMenu.xml)

catavolt replied:
I accepted the suggestion ;)

catavolt committed:
Update mediaportal/MediaPortal.Base/skin/DefaultWideHD/DialogMenu.xml

Co-authored-by: Andrew J.Swan <ajs.blackhole@gmail.com>

catavolt committed:
Changes to new Radio properties, new GFX, Bugfixing

Changes to new Radio properties, new GFX, Bugfixing

catavolt committed:
Merge branch 'MP1.38_Final_DWHD_Update' of https://github.com/MediaPortal/MediaPortal-1 into MP1.38_Final_DWHD_Update

catavolt committed:
Fixed bug in TVSeries

Fixed bug in TVSeries

catavolt committed:
Update GUIVideoTitle.cs

Reversed to Original

catavolt committed:
TVSeries bugfixes

TVSeries bugfixes

catavolt committed:
Bugfixes MyMusic

Bugfixes MyMusic: Fixed album views, position of music thumbs

catavolt committed:
A LOT of default GFX changes

A LOT of default GFX changes due to discussion in forum thread + a lot of enhanced xmls ;)

catavolt committed:
Lot of corrected xmls and GFX

Lot of corrected xmls and GFX

catavolt committed:
Bugfixes and new GFX

Bugfixes and new GFX

catavolt committed:
Update mediaportal/MediaPortal.Base/skin/DefaultWideHD/Translations/en-us.xml

Co-authored-by: Andrew J.Swan <ajs.blackhole@gmail.com>

andrewjswan committed:
Revert `MyVideo` changes

andrewjswan commented:
I reverted the changes to the `MyVideo` plugin.

PS: There seems to be some kind of conflict with the 1.39 branch.

catavolt committed:
Merge branch 'MP1.39_Final_Test1' into MP1.38_Final_DWHD_Update

catavolt commented:
Fixed the myhome and myhomeplugins :)

andrewjswan commented:
Will there be more changes? Or am I merging these?

catavolt commented:
Yup, missing the latest changes for the new radio xmls, but as a test whether all oprevious changes are in would be good ;)
THX!

andrewjswan commented:
> but as a test whether all oprevious changes are in would be good

Let's make all the changes, and then we'll close this PR.

catavolt commented:
Agree, but this will still take some time (a week or so) ;)

andrewjswan commented:
No problem—let me know when it's ready.

catavolt committed:
Last changes regarding new Radio functions

Last changes regarding new Radio functions :D

catavolt committed:
Delete radio.jpg

catavolt committed:
Merge branch 'MP1.38_Final_DWHD_Update' of https://github.com/MediaPortal/MediaPortal-1 into MP1.38_Final_DWHD_Update

catavolt committed:
Update Radio.jpg

catavolt commented:
Hi Andrew, latest files for new Radio functions added ;)
Ready to go with the new build 1.39 Final (we must still find a new name) ;)

catavolt committed:
Last changes for MyRadio ;)

Last changes for MyRadio ;)

catavolt committed:
Changed label for "Title"

Changed label for "Title" - Hope this was the last change for the new Radio functions!

andrewjswan commented:
Merged. If you need to make the following changes, please add them to this branch: https://github.com/MediaPortal/MediaPortal-1/tree/MP1.39_Final_DWHD_Update
